tests/queries/0_stateless/01060_substring_negative_size.sql

```diff
@@ -2,35 +2,115 @@ select substring('abcdefgh', 2, -2);
 select substring('abcdefgh', materialize(2), -2);
 select substring('abcdefgh', 2, materialize(-2));
 select substring('abcdefgh', materialize(2), materialize(-2));
+select substring(materialize('abcdefgh'), 2, -2);
```

Comment:
@RogerYK we prefer not to modify old tests, but create new ones.

Reply:
OK